QuizBowlDiscordScoreTracker/Web/wwwroot/js/main.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var connection = new signalR.HubConnectionBuilder().withUrl("/hub").build();
+var nameDisplay = document.getElementById("name");
+var sound1 = document.getElementById("sound1");
```

Comment:
Consider renaming sound1 to sound, since we don't have other sounds

Reply:
fixed in 0907327